### PR TITLE
LBAC-40: check and show the required Modules

### PR DESCRIPTION
**Pull request by VANKINEENITAWRUN**
## Issue I worked on

see https://issues.openmrs.org/browse/LBAC-40

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by HerbertYiga** [comment]
@VANKINEENITAWRUN  are you returning a json as required by the ticket?

**Comment by VANKINEENITAWRUN** [comment]
> @VANKINEENITAWRUN are you returning a json as required by the ticket?

gsocgit1
yes @HerbertYiga

**Comment by suthagar23** [comment]
Added comments in JIRA, and follow OpenMRS  JIRA guidelines also

**Comment by dkayiwa** [comment]
@VANKINEENITAWRUN did you see the above comments?

File: omod/src/main/java/org/openmrs/module/locationbasedaccess/web/controller/ModuleDependencyController.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@Controller
+@RequestMapping(value = "/rest/" + RestConstants.VERSION_1 + "/lbac/module-dependency")
+public class ModuleDependencyController extends BaseRestController {
+
+	/**
+	 * @return modules required for LBAC module installation
+	 */
+	@RequestMapping(method = RequestMethod.GET)
```
Comment (suthagar23) [inline omod/src/main/java/org/openmrs/module/locationbasedaccess/web/controller/ModuleDependencyController.java]: Can't we use getMapping?
https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/web/bind/annotation/GetMapping.html

Comment (VANKINEENITAWRUN) [reply omod/src/main/java/org/openmrs/module/locationbasedaccess/web/controller/ModuleDependencyController.java]: @suthagar23  we can use if we update the spring version to 4.3

**Pull request merged**